.gitignore

```diff
@@ -1,5 +1,7 @@
 # Created by https://www.gitignore.io/api/node
 
+*.swp
```

Comment:
@clindsey Could you set this setting at your `$HOME/.config/git/ignore` ?
[reference](https://git-scm.com/docs/gitignore)

You also can set gitignore for editor by below.
```
vim ~/.gitignore   # for global setting
git config --global core.excludesfile $HOME/.gitignore
```